index.html

```diff
@@ -16,6 +16,13 @@
     </header>
     <main>
       <section id="headline" class="headline-section flex-column">
+        <nav class="only-desktop">
+          <ul class="flex-row-header">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#about-me">About</a></li>
+            <li><a href="#contact-section">Contact</a></li>
+          </ul>
+        </nav>
```

Comment:
- [ ]  Your design is looking great! However, it is important to make a few changes to make your design look more similar to the Figma template:

Template:
![Captura de pantalla 2022-03-25 a la(s) 11 00 28 p  m](https://user-images.githubusercontent.com/83717285/160223985-31efd8b4-cdde-4736-b676-49041a85a320.png)


Your design:
![Captura de pantalla 2022-03-25 a la(s) 10 59 06 p  m](https://user-images.githubusercontent.com/83717285/160223987-01d584ce-6ce4-43af-b100-5162e76846e0.png)

Please place the languages list in just one line like the template.

Reply:
Done, for now, thank you for the observation.

Reply:
not resolved